validators/versions-denali.md

```diff
@@ -4,41 +4,75 @@ You can find the compatible versions in this page. Set up requires only Validato
 
 The rest of the versions were provided for docker containers and correspondent tags in their individual code repositories.
 
+### ** Prerequisites
+
+#### Skale node
+-   A Linux x86_64 machine
+-   SGX-enabled Intel processor
+-   Ports 22, 3009, 8080, 9100, and 10000–18192, and ICMP IPv4 open for all
+-   Ubuntu 18.04
+-   2TB attached storage
+-   8 core
+-   32GB RAM
+-   16GB swap
+
+
 ### Mainnet Versions for Set up
 
-**Node CLI version**: [1.0.0](https://github.com/skalenetwork/skale-node-cli/releases/download/1.0.0/skale-1.0.0-Linux-x86_64) 
+**Node CLI**: [1.1.0](https://github.com/skalenetwork/node-cli/releases/download/1.1.0/skale-1.1.0-Linux-x86_64) 
+
+**Validator CLI**:  [1.2.0](https://github.com/skalenetwork/validator-cli/releases/tag/1.2.0)
+
+**SGX Wallet**: [WIP](https://github.com/skalenetwork/sgxwallet/releases/tag/1.58.5-stable.1)
```

Comment:
this looks old

Reply:
@ebruoguzberk @badrogger  we're still waiting for the final version of SGX wallet, we can put 1.70.0-stable.0 for now and update if needed